packages/project-utils/bundling/function/buildFunction.js

```diff
@@ -54,4 +73,21 @@ module.exports = options => {
             resolve();
         });
     });
+
+    const handlerFile = fs.readFileSync(options.cwd + "/build/handler.js", {
+        encoding: "utf8",
+        flag: "r"
+    });
+    const includesGraphQl = handlerFile.includes("handler-graphql");
+
+    if (includesGraphQl) {
+        try {
+            await getLatestTelemetryCode;
```

Comment:
You mean `await getLatestTelemetryCode();` ?

Reply:
Let's rename this function. `get` prefix in function name usually indicates that the function is returning a value, but in this case, the function is really just adapting the manipulating built code.

Let's call it for example `injectHandlerTelemetry`.